### PR TITLE
docs: update changelog

### PR DESCRIPTION
**Pull request by slonka**
Signed-off-by: slonka <slonka@users.noreply.github.com>

### Summary

Update changelog.md before release

### Reason

Release

### Testing

Not needed

### CODE REVIEW

File: app/mesh/changelog.md [hunk 1]
```diff
@@ -4,13 +4,13 @@ no_version: true
...
-> Release on **TBD**
+> Release on **2022/08/22**
 
 Built on top of [Kuma 1.8.0](https://github.com/kumahq/kuma/releases/tag/1.8.0)
 
-### Changes
+### Upgrading
 
-- TO DO
+- add `VIEW_CLUSTERS` and `VIEW_STATS` to admin `AccessRole`
```
Comment (lena-larionova) [inline app/mesh/changelog.md:13]:
```suggestion
- Added `VIEW_CLUSTERS` and `VIEW_STATS` to admin `AccessRole`
```

**Comment by lahabana** [comment]
@slonka can we add the changelog for kong-mesh here too?

**Comment by slonka** [comment]
Is it copy paste of Kuma or just link to kuma? plus what changed in just KM?

File: app/mesh/changelog.md [hunk 2]
```diff
@@ -4,13 +4,23 @@ no_version: true
...
-> Release on **TBD**
+> Release on **2022/08/22**
 
 Built on top of [Kuma 1.8.0](https://github.com/kumahq/kuma/releases/tag/1.8.0)
 
 ### Changes
 
-- TO DO
+- Add "replace" function to CommonName template in CAs which support it (ACMPCA, cert-manager, Vault).
+- Improve error message when non-https URL is used with OPA.
```
Comment (bartsmykla) [inline app/mesh/changelog.md:14]: I don't think it's important enough to have it in this changelog to be honest

File: app/mesh/changelog.md [hunk 3]
```diff
@@ -4,13 +4,23 @@ no_version: true
...
+- Fix ZoneControlPlane token generation by setting access type to RBAC in the generated default.
+- Improve RBAC logic by checking both old and new spec on updates.
+- Change log level of RBAC to `debug` by default except the result of RBAC validation.
```
Comment (bartsmykla) [inline app/mesh/changelog.md:17]: Same as above

File: app/mesh/changelog.md (hunk 2 above)
Comment (slonka) [reply app/mesh/changelog.md]: removed

File: app/mesh/changelog.md (hunk 3 above)
Comment (slonka) [reply app/mesh/changelog.md]: removed

File: app/mesh/changelog.md [hunk 4]
```diff
@@ -4,13 +4,21 @@ no_version: true
 ---
 
 ## 1.9.0
-> Release on **TBD**
+> Release on **2022/08/22**
```
Comment (bartsmykla) [inline app/mesh/changelog.md:7]:
```suggestion
> Release on **2022/08/23**
```

**Review by bartsmykla — APPROVED** [review]
lgtm

**Comment by lahabana** [comment]
Would the Changelog tool from Kuma be helpful there? AFAIK it's repo agnostic

**Comment by slonka** [comment]
> Would the Changelog tool from Kuma be helpful there? AFAIK it's repo agnostic

There are relatively small number of PRs there so I just manually

```
| grep -v "auto upgrade Kuma" | grep -iv "Bump" | grep -v "upgrade dependencies" |
```

the git log

**Pull request merged**